package.json

```diff
@@ -6,7 +6,7 @@
     "start": "node server/index.js",
     "server": "nodemon server/index.js",
     "client": "cd client && npm start",
-    "dev": "concurrently \"npm run server\" \"npm run client\"",
+    "dev": "concurrently --kill-others-on-fail \"npm run server\" \"npm run client\"",
```

Comment:
is it like fuser -k 3000/tcp?

Reply:
i'm not sure what ```fuser -k 3000/tcp``` does!
basically this script means that if either of the scripts that are running concurrently fails, it will shut down everything.

for example, if you were running ```npm run dev``` and the client script failed, it would also shut down the server. :)

Reply:
```fuser -k 3000/tcp ``` it kill the port